snafu/benchmarks/nighthawk/Dockerfile

```diff
@@ -0,0 +1,11 @@
+ARG BASE_IMAGE=quay.io/cloud-bulldozer/nighthawk-base:latest
```

Comment:
What is in here/where is the docker file for it?

Reply:
It's just a copy of binaries from my local pod file system to image. So I did not include it. But the image is present in the cloud-bulldozer already. For more info on how those binaries are generated: https://github.com/envoyproxy/nighthawk

Reply:
resolving this comment and will use the identical one that  Murali stated for discussion.